testing/test_skipping.py

```diff
@@ -861,9 +861,25 @@ def test_hello():
                 pass
         """
         )
-        result = pytester.runpytest("-rs")
+        result = pytester.runpytest("-rs", "--strict-markers")
```

Comment:
Not directly related, see https://github.com/pytest-dev/pytest/pull/2552#discussion_r125535869